### ISSUE TITLE
The silent miniconda setup crashes Outlook on Windows

### ISSUE THREAD

**Issue by davidanthoff** [Luthaf/Conda.jl#15]
This is really bad, because it can lead to real data loss in Outlook (for example if you have composed a long email but haven't saved/sent it yet). I reported the problem previously at conda/conda#1084, but no reaction there. The issue in the conda repo also points to some discussion of this bug on some mailing list that others have encountered, the root problem seems to be an NSIS problem.

@stevengj This now also affects PyPlot and IJulia, both packages now crash Outlook when you add them (i.e. when the Conda build.jl script runs). When I previously worked on my version of a private Python setup for IJulia, that was the point when everyone agreed that this was unacceptable behavior, and then I worked on a setup that doesn't use miniconda, but a private Python install and used pip instead. I think at this point both IJulia and PyPlot should probably drop the dependency on Conda until this issue is fixed.

**Comment by stevengj** [comment]
As of #14, miniconda now no longer installs on `Pkg.build("Conda")`, but only if Conda is actually needed to install something.   So, people who already have Python are not affected by this.

**Comment by davidanthoff** [comment]
Ah, that is good. Lets hope this can get fixed upstream, it still is a nasty bug.

**Comment by Luthaf** [comment]
> it still is a nasty bug.

Yes it is! I can disable the silent install for now.

**Comment by stevengj** [comment]
If you disable silent install, does that mean that users have to click through some dialogs?  If they make the wrong choices, will it screw up the install?

**Comment by Luthaf** [comment]
I do not know. I'll have to test.

**Comment by davidanthoff** [comment]
Another option might be to test whether the Outlook.exe process is running before the miniconda process is started, and if it is running, just throw an error with a message that users should shutdown Outlook and then repeat the last operation. And then hope that they fix this upstream and then remove this check again.

Checking is probably easiest done using Powershell, the command ``Get-Process Outlook`` should probably do it, I think one would just have to check whether that throws an error (not running) or returns something (running).

**Comment by stevengj** [comment]
There is a [win32 API to get the list of running processes](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682623(v=vs.85).aspx), but I agree that PowerShell seems a lot easier.  

Presumably, this check only needs to be in `Conda._install`.

**Comment by Luthaf** [comment]
How can we call powershell from Julia? I am not a windows expert at all ...

**Comment by davidanthoff** [comment]
I think something like this should work

````julia
try
  run(`powershell "Get-Process Outlook"`)
  println("Outlook is running")
catch e
  println("Outlook is not running")
end
````

Obviously you need to adjust a bit, but the general theme should be this.

**Comment by Luthaf** [comment]
@davidanthoff this is implemented in the windows branch, in commit dba92625e54c6f4c0f0942103a581334c95bedf8. Can you give it a look and test whether the error is issued if Outlook is running?
